src/program/lwaftr/virt/ports/lwaftr1/b.cfg

```diff
@@ -1,5 +1,6 @@
 return {
   { port_id = "1B",
-    disable_mergeable_rx_buffer = true,
```

Comment:
This was an outdated flag that originally was used to initialize VirtioNetDevice with mergeable_rx_buffer disabled https://github.com/Igalia/snabb/blob/lwaftr/src/lib/virtio/net_device.lua#L72

It was set to true in our configuration files but the flag is actually not being used.